packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/new/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dedicatedCloud_datacenter_backup_new_description1": "Il semblerait que vous n’avez pas activé de service de sauvegarde",
+  "dedicatedCloud_datacenter_backup_new_description2": "Vous pouvez choisir entre 3 nouvelles offres pour protéger vos machines virtuelles (VM) : <strong>Standard</strong>, <strong>Advanced</strong> ou <strong>Premium</strong>.",
+  "dedicatedCloud_datacenter_backup_new_tariff_url": "Consulter en détail la nouvelle gamme Veeam Backup",
+  "dedicatedCloud_datacenter_backup_new_choose_offer": "Choix de la solution de sauvegarde",
+  "dedicatedCloud_datacenter_backup_new_choose_offer_action": "Choisir l'offre",
+  "dedicatedCloud_datacenter_backup_new_validate_conditions": "Validation des conditions d’utilisation",
+  "dedicatedCloud_datacenter_backup_new_activate_offer_action": "Activer l’offre",
+  "dedicatedCloud_datacenter_backup_new_create_success": "Votre offre a bien été activée. Vous êtes maintenant abonné·e à la solution Veeam Backup {{ offerType }}. Si vous souhaitez changer d’offre, merci d’attendre 15 jours pour tout autre changement.<br/><a href='{{ guideLink }}' target='_blank' class='oui-link_icon'>Consulter le guide sur Veeam Backup afin de configurer votre sauvegarde <span class='oui-icon oui-icon-arrow-right_extra-thin' aria-hidden='true'></span></a>",
```

Comment:
Please split into two translations instead, `br` has a semantic meaning: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

Reply:
The link was not required. Hence it has been removed. Thanks